lib/Settings/Admin.php

```diff
@@ -21,10 +21,19 @@
 
 namespace OCA\GroupFolders\Settings;
 
+use OCA\GroupFolders\Service\DelegationService;
 use OCP\AppFramework\Http\TemplateResponse;
 use OCP\Settings\ISettings;
 
 class Admin implements ISettings {
+
+	/** @var DelegationService */
+	private $delegationService;
+
+	public function __construct(DelegationService $delegationService) {
+		$this->delegationService = $delegationService;
```

Comment:
Seems to be unused.

Reply:
Yes, right. That's a leftover that should have been deleted during commit 1ab30bb2131a720a8b490dfe33f95901b448af87

Reply:
removed